tsconfig.build.json

```diff
@@ -13,7 +13,7 @@
     }
   },
   "angularCompilerOptions": {
-    "enableIvy": false,
+    "enableIvy": true,
```

Comment:
Should we be compiling with ivy enabled?

Reply:
Nope, Ivy code shoudn't be published as it won't be compatible with VE.

Reply:
Good point